Mathlib/Data/Fintype/Lattice.lean

```diff
@@ -67,3 +67,19 @@ theorem Finite.exists_min [Finite α] [Nonempty α] [LinearOrder β] (f : α →
   cases nonempty_fintype α
   simpa using exists_min_image univ f univ_nonempty
 #align finite.exists_min Finite.exists_min
+
+theorem iSup_finsetUniv [CompleteLattice α] [Fintype β] {f : β → α} :
+    ⨆ x ∈ Finset.univ, f x = ⨆ x, f x :=
+  by simp
+
+theorem iInf_finsetUniv [CompleteLattice α] [Fintype β] {f : β → α} :
+    ⨅ x ∈ Finset.univ, f x = ⨅ x, f x :=
+  iSup_finsetUniv (α := αᵒᵈ)
+
+theorem Set.biUnion_finsetUniv [Fintype β] {u : β → Set α} :
```

Comment:
This doesn't feel necessary because we also have [Set.iSup_eq_iUnion](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Lattice.html#Set.iSup_eq_iUnion)

Reply:
I took this idea from [`Set.biInter_univ`](https://github.com/leanprover-community/mathlib4/blob/8666bd82efec40b8b3a5abca02dc9b24bbdf2652/Mathlib/Data/Set/Lattice.lean#L1081-L1082). I guess it depends on whether it's standard to [copy lemmas through definitions](https://github.com/leanprover-community/mathlib4/blob/8666bd82efec40b8b3a5abca02dc9b24bbdf2652/Mathlib/Data/Finset/Lattice.lean#L2126-L2174) since `simp`/`rw` don't rewrite through defeq.

But yes, one can currently do:

```lean
theorem Set.biUnion_finsetUniv [Fintype β] {u : β → Set α} :
    ⋃ x ∈ Finset.univ, u x = ⋃ x, u x := by
  unfold iUnion
  rw [← Finset.sup_eq_iSup, Finset.sup_univ_eq_iSup]
  -- or: simp_rw [← Set.iSup_eq_iUnion, ← Finset.sup_eq_iSup, Finset.sup_univ_eq_iSup]
```